src/plugins/intel_gpu/src/plugin/transformations/rms_fusion.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "rms_fusion.hpp"
+
+#include "intel_gpu/op/rms.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/power.hpp"
+#include "openvino/op/reduce_mean.hpp"
+#include "openvino/op/sqrt.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+namespace ov {
+namespace intel_gpu {
+
+RMSFusion::RMSFusion() {
+    using namespace ov::pass::pattern;
+
+    // Detect RMS decomposition pattern
+    //  x * 1/Sqrt(ReduceMean(x^2,axes)+eps) * gamma
+    auto x = any_input();
+
+    // x^2
+    auto const_2 = wrap_type<ov::op::v0::Constant>();
+    auto power = wrap_type<ov::op::v1::Power>({x, const_2});
+
+    // ReduceMean(x^2,axes)
+    auto mean_axes = wrap_type<ov::op::v0::Constant>();
+    auto mean = wrap_type<ov::op::v1::ReduceMean>({power, mean_axes});
+
+    // ReduceMean(x^2,axes)+eps
+    auto eps = wrap_type<ov::op::v0::Constant>();
+    auto add_eps = wrap_type<ov::op::v1::Add>({mean, eps});
+
+    // Sqrt(ReduceMean(x^2,axes)+eps)
+    auto sqrt = wrap_type<ov::op::v0::Sqrt>({add_eps});
+
+    // 1/Sqrt(ReduceMean(x^2,axes)+eps)
+    auto const_minus_1 = wrap_type<ov::op::v0::Constant>();
```

Comment:
Where do you check the values of these constants? I suppose you need to add a predicate like [here](https://github.com/openvinotoolkit/openvino/blob/8020530e678f875e8ac35162522e4bf174921806/src/common/transformations/src/transformations/common_optimizations/mvn_fusion.cpp#L28) and change constants wrappers to something like 
`pattern::wrap_type<ov::op::v0::Constant>(value_is_equal_to<float>({-1.f}));`

Reply:
Updated to check single value for constant wrappers